Docker/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM oraclelinux:8
 
 LABEL author="OPERA ADT" \
-    description="RTC gamma release R3" \
-    version="0.4-calval"
+    description="RTC gamma release R4" \
+    version="0.4-gamma"
```

Comment:
```suggestion
    description="RTC cal/val release R4" \
    version="0.4-calval"
```

Reply:
Thanks @gshiroma. I was about to ask you about the label ;)